dev/wlp-jakartaee-transform/rules/jakarta-faces-initialization.properties

```diff
@@ -1,7 +1,4 @@
 #com.ibm.ws.myfaces.initializer.xml
 javax.servlet.ServletContainerInitializer=jakarta.servlet.ServletContainerInitializer
-#com.ibm.ws.org.apache.myfaces_3_0_1_0.ResourceProvider.xml
+#com.ibm.ws.org.apache.myfaces_1_0.ResourceProvider.xml
```

Comment:
Can you explain the versioning here? I assume this is due to this change:

  com.ibm.ws.org.apache.myfaces_3_0_${replace;${bVersion};\\.;_}.ResourceProvider; \
  io.openliberty.org.apache.myfaces_${replace;${bVersion};\\.;_}.ResourceProvider; \

Reply:
Just to keep this generic as you already know.. I removed 3_0 as this applies to 2.3 & 3.0.  I'm not sure why the 1_0 (bVersion)  is needed there but I kept it here since it was there before.